packages/cardpay-reward-programs/scripts/read_parquet.py

```diff
@@ -0,0 +1,15 @@
+import pandas as pd
+import typer
+from cloudpathlib import AnyPath
+
+
+def read_parquet(reward_program_id: str, payment_cycle: int):
```

Comment:
Would be nice to default to staging but allow either staging or prod. Not a blocker.